diff --git a/shared-module/usb_hid/__init__.c b/shared-module/usb_hid/__init__.c
index 4d0d5fc..3bcec8d 100644
--- a/shared-module/usb_hid/__init__.c
+++ b/shared-module/usb_hid/__init__.c
@@ -45,7 +45,7 @@ static const uint8_t usb_hid_descriptor_template[] = {
 #define HID_IN_ENDPOINT_INDEX (20)
     0x03,        // 21 bmAttributes (Interrupt)
     0x40, 0x00,  // 22,23  wMaxPacketSize 64
-    0x08,        // 24 bInterval 8 (unit depends on device speed)
+    0x01,  // 24 bInterval 1 (unit depends on device speed)
 
     0x07,        // 25 bLength
     0x05,        // 26 bDescriptorType (Endpoint)
@@ -53,7 +53,7 @@ static const uint8_t usb_hid_descriptor_template[] = {
 #define HID_OUT_ENDPOINT_INDEX (27)
     0x03,        // 28 bmAttributes (Interrupt)
     0x40, 0x00,  // 29,30 wMaxPacketSize 64
-    0x08,        // 31 bInterval 8 (unit depends on device speed)
+    0x01,  // 31 bInterval 1 (unit depends on device speed)
 };
 
 #define MAX_HID_DEVICES 8
